decidim-core/lib/decidim/download_your_data_serializers/download_your_data_user_serializer.rb

```diff
@@ -44,11 +40,15 @@ def serialize
           confirmed_at: resource.confirmed_at,
           confirmation_sent_at: resource.confirmation_sent_at,
           unconfirmed_email: resource.unconfirmed_email,
-          delete_reason: resource.delete_reason,
-          deleted_at: resource.deleted_at,
           managed: resource.managed,
           officialized_at: resource.officialized_at,
-          officialized_as: resource.officialized_as
+          officialized_as: resource.officialized_as,
+          roles: resource.roles,
+          accepted_tos_version: resource.accepted_tos_version,
+          type: resource.type,
```

Comment:
I am not 100% convinced that this field should be present here. Most likely the CSV will have only 1 row, and the value of this field is always "Decidim::User". 
As far as i can see, the groups are residing in their own file. 
```suggestion
```

Reply:
You're absolutely right !

Also worth mentioning is that with https://github.com/decidim/decidim/issues/13969 the idea is to fully remove the Groups and merge it with Users, so it'll even have less sense